pkg/cloudprovider/aws/amifamily/ami.go

```diff
@@ -39,7 +39,7 @@ func (p *AMIProvider) Get(ctx context.Context, instanceType cloudprovider.Instan
 	}
 	output, err := p.ssm.GetParameterWithContext(ctx, &ssm.GetParameterInput{Name: aws.String(ssmQuery)})
 	if err != nil {
-		return "", fmt.Errorf("getting ssm parameter, %w", err)
+		return "", fmt.Errorf("getting ssm parameter with query=%s, %w", ssmQuery, err)
```

Comment:
Technically the query IS the parameters' name
```suggestion
		return "", fmt.Errorf("getting ssm parameter %s, %w", ssmQuery, err)
```
```suggestion
		return "", fmt.Errorf("getting ssm parameter with query=%s, %w", ssmQuery, err)
```